canonical/maas-ui
# chore: Clean up legacy header MAASENG-1312

## Done

- Removed legacy `Header` component and references
- Added `is-pinned` class for `l-navigation-bar` to be shown when logged out

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Go to UI
- Ensure nav displays normally at all breakpoints
- Log out
- Ensure header is shown correctly at all breakpoints

## Fixes

Fixes [MAASENG-1312](https://warthogs.atlassian.net/browse/MAASENG-1312)

[MAASENG-1312]: https://warthogs.atlassian.net/browse/MAASENG-1312?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
